tentacoli.js

```diff
@@ -291,6 +294,21 @@ Tentacoli.prototype.request = function (data, callback) {
   return this
 }
 
+Tentacoli.prototype.fire = function (data, callback) {
+  var that = this
+  var req = new Request(null, callback)
+
+  try {
+    wrapStreams(that, data, req, true)
+  } catch (err) {
+    return this
```

Comment:
Personally I'll go for `b`, for two reasons:
- The user does not need to write try catch block every time, very annoying.
- The function is a fire and forget, this means that the user do not care about the result, otherwise it should use a standard request.

Thoughts?